app/controllers/experts/avis_controller.rb

```diff
@@ -147,7 +147,7 @@ def bilans_bdf
     end
 
     def telecharger_pjs
-      files = ActiveStorage::DownloadableFile.create_list_from_dossiers(Dossier.where(id: @dossier.id))
+      files = ActiveStorage::DownloadableFile.create_list_from_dossiers(Dossier.where(id: @dossier.id), include_avis_for_expert: current_expert)
```

Comment:
```suggestion
      files = ActiveStorage::DownloadableFile.create_list_from_dossiers(Dossier.where(id: @dossier.id), include_avis_for_expert: expert_signed_in?)
```

Ça m'a pris une minute pour comprendre que pourquoi tu passai un expert comme valeur d'un flag.